README.md

```diff
@@ -141,12 +141,22 @@ docker exec db2 su - db2inst1 -c "db2 CONNECT TO onprem USER watson USING secret
 
 At this point, you have a Db2 database instance loaded with sample data.
 
-Before you proceed further, you also need to take note of your workstation's LAN IP. You can
-find that address using:
+Before you proceed further, you also need to take note of your workstation's
+LAN IP. You can find that address using either `hostnam -I` or `ifconfig`,
```

Comment:
s/hostnam/hostname/

Reply:
That didn't work for me on my desktop:

    $ python -c "import socket; print(socket.gethostbyname(socket.gethostname()))"
    127.0.1.1

(loopback addresses won't work for this use case)

Reply:
I've been trying to figure out where the `127.0.1.1` came from in the above output. It turns out, I have a manual entry for `127.0.1.1 myhostname` in `/etc/hosts`, in addition to `127.0.0.1 localhost`.